### ISSUE TITLE
Allow to pass more span metadata to `span` with block

### ISSUE THREAD

**Issue by sekiyama58** [spandex-project/spandex#73]
Currently metadata is only allowed in `update_span` arguments.
For simplicity, it would be nice if we can pass metadata arguments to `Tracer.span` with block, like
```
    Tracer.span("span_me_also", resource: "aaa", sql_query: [query: "..."]) do
      ...
    end
```

**Comment by zachdaniel** [comment]
AFAIK this is how it works. You should be able to pass in any data on creation that you can on update_span.

**Comment by zachdaniel** [comment]
Have you confirmed that the metadata set that way is not honored?

**Comment by sekiyama58** [comment]
Thanks for response.
I have tried again with v2.1.0 and the issue was not confirmed.
It seems I have been confused by dynamic configuration issue that is fixed by 2.1.0 release.
So I'm closing this issue now.

Thanks!
